07.bowling_object/lib/game.rb

```diff
@@ -0,0 +1,69 @@
+# frozen_string_literal: true
+
+require_relative 'frame'
+
+class Game
+  def initialize(marks)
+    @marks = marks.split(',')
+  end
+
+  def divide_into_rolls(marks)
+    rolls = []
+    9.times do
+      roll = marks.shift(2)
+      if roll.first == 'X'
+        marks.unshift(roll.last)
+        roll = ['X']
```

Comment:
```suggestion
        marks.unshift(roll.pop)
```

にすれば少しは効率よさそうですね
https://docs.ruby-lang.org/ja/latest/method/Array/i/pop.html